sdk/framework/path.go

```diff
@@ -173,11 +173,18 @@ type DisplayAttributes struct {
 	// Navigation indicates that the path should be available as a navigation tab
 	Navigation bool `json:"navigation,omitempty"`
 
+	// ItemType is the type of item this path operates on
+	ItemType string `json:"itemType,omitempty"`
+
 	// Group is the suggested UI group to place this field in.
 	Group string `json:"group,omitempty"`
 
 	// Action is the verb to use for the operation.
 	Action string `json:"action,omitempty"`
+
+	// EditType is the type of form field needed for a property
+	// e.g. "textarea" or "file"
+	EditType string `json:"editType,omitempty"`
```

Comment:
these are all of the available keys for the `DisplayAttributes` map. see the full list here:

https://github.com/hashicorp/vault/blob/master/sdk/framework/path.go#L169-L196